src/main/java/core/basesyntax/FigureSupplier.java

```diff
@@ -0,0 +1,39 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class FigureSupplier {

```

Comment:
Lets create constants for all magic numbers and for "White"

Reply:
did not understand about magic numbers)

Reply:
In these way - all numbers

Reply:
In these way - all numbers